src/xdocs/config_metrics.xml

```diff
@@ -101,6 +101,23 @@
         <source>
 &lt;module name=&quot;BooleanExpressionComplexity&quot;/&gt;
         </source>
+        <p>Code Example:</p>
+        <source>
+public class Test
+{
+  public static void main(String ... args)
+  {
+    boolean a = true;
+    boolean b = false;
+
+    boolean c = (a &amp; b) | (b &amp; a);
+    //OK, (&amp;,|) are counted as 1 which gives the total complexity of 3(max allowed 3)
```

Comment:
1 space after all comments, it makes them easier to read.

> `(&amp;,|)`

I would not use parenthesis or commas as it could be confused with the actual parenthesis in the code. You can use the technical name for the operators or stick with `&` and `|` but again, space them apart so they aren't so compact. The comment can span multiple lines if needed.

> total complexity of 3(max allowed 3)

Again, space between words and `(`.

Reply:
Done